spec/src/main/doc/injectionelresolution.asciidoc

```diff
@@ -384,16 +375,16 @@ When the container creates a new instance of a managed bean, session bean, or of
 
 The container must ensure that:
 
-* Initializer methods declared by a class X in the type hierarchy of the bean are called after all injected fields declared by X or by superclasses of X have been initialized, and after all Java EE component environment resource dependencies declared by X or by superclasses of X have been injected.
-* Any +@PostConstruct+ callback declared by a class X in the type hierarchy of the bean is called after all initializer methods declared by X or by superclasses of X have been called, after all injected fields declared by X or by superclasses of X have been initialized, and after all Java EE component environment resource dependencies declared by X or by superclasses of X have been injected.
-* Any servlet +init()+ method is called after all initializer methods have been called, all injected fields have been initialized and all Java EE component environment resource dependencies have been injected.
+* Initializer methods declared by a class X in the type hierarchy of the bean are called after all injected fields declared by X or by superclasses of X have been initialized, and after all component environment resource dependencies declared by X or by superclasses of X have been injected.
```

Comment:
Is the container supposed to support resource injection or not? The removal of resource producers seems to indicate that not but this wording seems to indicate that it is supported.